web/modules/custom/mymodule/src/Controller/FirstController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Drupal\mymodule\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+
+/**
+ * Created a controller for testing.
+ */
+class FirstController extends ControllerBase {
+
+  /**
+   * A Simple Controller
+   *
+   * @return array
+   *   Returns the render array.
+   */
+  public function simpleContent() {
+    return [
+      '#type' => 'markup',
+      '#markup' => t('Hello World i am from the first Controller'),
+    ];
+  }
+
+  /**
+   * Function to render the variable content.
+   * 
+   * @param string $name1
+   *   A String.
```

Comment:
The comment should be more precise. Try to explain more like what is the purpose of the string etc

Reply:
I don't see these changes getting implemented yet